packages/server-boilerplate/src/routes/TranslatableRoute.ts

```diff
@@ -16,9 +16,10 @@ type StringKey = {
 
 type ObjectKey = {
   type: 'object';
-  properties: { [key: string]: TranslationKey };
+  properties?: { [key: string]: TranslationKey };
   // Keys within the object to translate
-  keys?: 'all' | string[];
+  keysToTranslate?: 'all' | string[];
```

Comment:
I'm torn between the very clear `'all'` and just reusing `'*'`, which is showing up in the report server in lots of places that mean "all" as well as "all others". What do you think?